### DISCUSSION TITLE
Does the latest "develop" branch support capillaryPressure in co2 injection case?

### DISCUSSION THREAD

**Discussion by wdmhouston** [GEOS-DEV/GEOS#1478] [Q&A]
I notice that  there are some modules in /src/coreComponents/constitutive/capillaryPressure folder, and I am wondering if capillaryPressure in co2 injection case.   If yes, can you share any example?

**Comment by paveltomin** [comment]
It does, see https://geosx-geosx.readthedocs-hosted.com/en/latest/coreComponents/constitutive/docs/CapillaryPressureModels.html
